modules/ROOT/pages/tutorials/private-link.adoc

```diff
@@ -0,0 +1,120 @@
+[[aura-private-link-tutorial]]
+= Private link tutorial
+
+label:AuraDB-Virtual-Dedicated-Cloud[]
+
+The technical documentation for this section is applicable to all clouds (cloud agnostic).
+
+We will use the term “private link” to refer to:
+
+* Private Service Connect = Google Cloud platform
+* PrivateLink = AWS
+* Private Link = Azure
+
+In this tutorial, we’ll walk through the process of setting up a private link connection to securely connect your application to an AuraDB Virtual Dedicated Cloud environment. 
+A private link provides secure network connectivity between your application and AuraDB without exposing traffic to the public internet.
+Refer to Figure 1 for a visual representation of the network package traffic flow.
+
+.This diagram illustrates the network package traffic flow when an application connects to an Aura instance.
+image::aura-private-link-architecture.png[Fig.1 Aura Private Link/Private Service Connect Architecture]
+
+== Steps
```

Comment:
These steps are not steps that the user can take and maybe not necessary in a tutorial. They could move to another section where we would talk about the internals of private links.